### PR TITLE
Update to 1.8.0

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,7 +17,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
+  skip: True  # [unix and not py35]
```
Comment (isuruf) [inline recipe/meta.yaml:21]: I've added this as we don't need to build for different versions of python in unix. Python is merely used for running the test suite.

Comment (scopatz) [reply recipe/meta.yaml:21]: If that is true then Python shouldn't be part of the build requirements.

Comment (isuruf) [reply recipe/meta.yaml:21]: There was an error in the last statement. Python is used for building the test suite and hence used in the build requirement.

Comment (isuruf) [reply recipe/meta.yaml:21]: Test suite is built at the same time the library is built, so I cannot build the tests in a later stage.

Comment (scopatz) [reply recipe/meta.yaml:21]: I see.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**